test/default_relay_host.bats

```diff
@@ -2,7 +2,7 @@ load 'test_helper/common'
 
 function setup() {
     docker run -d --name mail_with_default_relay \
-		-v "`pwd`/test/config/relay-hosts":/tmp/docker-mailserver \
+		-v "$(duplicate_config_for_container relay-hosts/)":/tmp/docker-mailserver \
```

Comment:
Not sure whether the `/` in the end makes a difference here. Could you double check?

Reply:
I removed the / but since this was a directory, I don't see much reason why this should be problematic